lldb/source/Target/TargetProperties.td

```diff
@@ -201,6 +201,10 @@ let Definition = "target" in {
   def DebugUtilityExpression: Property<"debug-utility-expression", "Boolean">,
     DefaultFalse,
     Desc<"Enable debugging of LLDB-internal utility expressions.">;
+  def LaunchWorkingDir: Property<"launch-working-dir", "String">,
+    DefaultStringValue<"">,
+    Desc<"A default value for the working directory to use when launching processes. "
+         "It's not used when empty.">;
```

Comment:
Here we should say how it interacts with commands that can set the working dir, and that changing this at runtime does not change the running process' working dir.

"This setting is only used when the target is launched. If you change this setting, the new value will only apply to subsequent launches."